src/olcs/OLCesium.ts

```diff
@@ -342,15 +290,19 @@ class OLCesium {
     this.lastFrameTime_ = frameTime;
 
     const julianDate = this.time_();
+    // initializeFrame private property
+    // @ts-ignore
```

Comment:
I think it is possible to reduce to the private access rule:
```suggestion
    // @ts-ignore TS2341
```